staking_deposit/cli/generate_bls_to_execution_change.py

```diff
@@ -177,12 +192,19 @@ def generate_bls_to_execution_change(
     )
 
     # Check if the given old bls_withdrawal_credentials is as same as the mnemonic generated
-    for i, credential in enumerate(credentials.credentials):
-        try:
-            validate_bls_withdrawal_credentials_matching(bls_withdrawal_credentials_list[i], credential)
-        except ValidationError as e:
-            click.echo('\n[Error] ' + str(e))
-            return
+    with click.progressbar(length=len(credentials.credentials), label=load_text(['msg_credentials_verification']),
+                           show_percent=False, show_pos=True) as bar:
+        executor_kwargs = [{
+            'credential': credential,
+            'bls_withdrawal_credentials': bls_withdrawal_credentials_list[i],
+        } for i, credential in enumerate(credentials.credentials)]
+
+        with concurrent.futures.ProcessPoolExecutor() as executor:
+            for e in executor.map(_validate_credentials_match, executor_kwargs):
+                bar.update(1)
+                if e is not None:
+                    click.echo('\n[Error] ' + str(e))
```

Comment:
Incredibly minor but can we add one more newline here?  I'd prefer a little whitespace for the error.
```
Creating your keys:		  [####################################]  5/5
Verifying your withdrawal credentials.  [#######-----------------------------]  1/5
[Error] The given withdrawal credentials does not match the old BLS withdrawal credentials that mnemonic generated.
```

Reply:
I'm happy to add an additionnal newline before `[Error]` to make it clear in the CLI.